consensus/src/threshold_simplex/config.rs

```diff
@@ -15,23 +13,22 @@ use std::{num::NonZeroUsize, time::Duration};
 /// Configuration for the consensus engine.
 pub struct Config<
     C: Signer,
+    S: SigningScheme,
     B: Blocker<PublicKey = C::PublicKey>,
-    V: Variant,
     D: Digest,
     A: Automaton<Context = Context<D>>,
     R: Relay,
-    F: Reporter<Activity = Activity<V, D>>,
-    S: ThresholdSupervisor<
-        Index = View,
-        Identity = V::Public,
-        Seed = V::Signature,
-        PublicKey = C::PublicKey,
-        Share = group::Share,
-    >,
+    F: Reporter<Activity = Activity<S, D>>,
 > {
     /// Cryptographic primitives.
     pub crypto: C,
 
+    /// List of validators for the consensus engine.
+    pub participants: Vec<C::PublicKey>,
+
+    /// Signing scheme for the consensus engine.
+    pub signing: S,
```

Comment:
dumb idea: seems like we should be able to just provide `S` and derive `C` + `C::PublicKey`

Reply:
Yes, I also thought about the signing scheme subsuming the participants, so that we have a single provider for everything. But it would increase the surface of `SigningScheme` and potentially more duplication across implementations. If you don't find this to be urgent can look into it in a follow-up.

Reply:
Logged https://github.com/commonwarexyz/monorepo/issues/1923.